### DISCUSSION TITLE
What's wrong with this?

### DISCUSSION THREAD

**Discussion by zhucan** [louwrentius/fio-plot#122] [Q&A]
fio-plot show me "/usr/local/lib/python3.6/site-packages/fio_plot/fiolib/supporting.py:466: UserWarning: Tight layout not applied. The left and right margins cannot be made large enough to accommodate all axes decorations. 
  plt.tight_layout(rect=[0, 0, 1, 1])", why?

**Comment by louwrentius** [comment]
To be frank, I've seen this message when creating a 3D graph and I don't know why exactly this happens, but since it's a warning I never cared about it, since the image always looked good to me.

**Reply by zhucan** [reply]
I will test again and  show the cmd and result
